test_ghub.py

```diff
@@ -1,15 +1,15 @@
 import os
 import unittest
 from io import StringIO
-from mock import patch
+from unittest.mock import patch
```

Comment:
With this you probably don't need `test_requirements.txt` might need to also update the build plan.

Reply:
Fixed, thanks for catching that.